### PR TITLE
Update docs on updating tokens and on ckan

### PR DESCRIPTION
**Pull request by BeckaL**
Updating a couple of things with some helpful stuff from 2nd line this week.

### CODE REVIEW

File: source/manual/alerts/expiring-api-tokens-in-signon.html.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ for Publishing API expires in X days*. In this example, the API User is
 - Copy the new token and prepare to replace it in [govuk-secrets].
 
 > How to do the last step depends on the application, but it should be
-> something like `rake eyaml:edit[integration,apps]`, depending on the
+> something like `rake eyaml:edit[integration,apps]` within the `puppet_aws` directory, depending on the
```
Comment (1pretz1) [inline source/manual/alerts/expiring-api-tokens-in-signon.html.md:37]: We could also suggest the non 'apps' eyaml as a potential place to look: `rake eyaml:edit[integration]`.

Comment (jyoung-gds) [reply source/manual/alerts/expiring-api-tokens-in-signon.html.md:37]: tiny thing, maybe include the single quotes required in eyaml (i.e. `rake 'eyaml:edit[integration,apps]'`)

File: source/manual/alerts/expiring-api-tokens-in-signon.html.md [hunk 2]
```diff
@@ -60,7 +61,13 @@ for Publishing API expires in X days*. In this example, the API User is
...
 > The api call could be something like
 >
-> `GdsApi::SupportApi.new(Plek.new.find("support-api"), bearer_token: ENV["SUPPORT_API_BEARER_TOKEN"])`
+> `gds_api = GdsApi::Router.new(Plek.new.find("router-api"), bearer_token: ENV["ROUTER_API_BEARER_TOKEN"])`
+>
+> `gds_api.get_route("/foo")`
+>
+> If the token works, then the above should get us something other than an unauthorised error.
+> You could test this out by setting up `gds_api` with an incorrect token, e.g. `bearer_token: "NOT_THE_REAL_TOKEN"`
+> and checking that this gives a different response to when it is set up with the new token.
```
Comment (jyoung-gds) [inline source/manual/alerts/expiring-api-tokens-in-signon.html.md:70]: I feel like revoking the old token should be the final step after this one, and this step should be more directional, the words "once you're happy" seem too vague (i.e. check the new token works with this step --> revoke the old token). Also maybe worth putting in the bit about waiting for Puppet to perpetuate the token before revoking it?

**Review by 1pretz1 — APPROVED** [review]
Looks good! 🍏

**Pull request merged**